spec/spec_helper.rb

```diff
@@ -38,7 +38,6 @@
     Puppet.settings[:strict] = :warning
   end
   c.filter_run_excluding(bolt: true) unless ENV['GEM_BOLT']
-  c.mock_with :rspec
```

Comment:
This is causing the unit test failures on appveyor. See https://github.com/puppetlabs/pdk-templates#specspec_helperrb on how to set this so that a `pdk update` keeps it around.